samples/dashboards/demo/personal-portfolio/demo.js

```diff
@@ -0,0 +1,579 @@
+        lastHoldingTotal = getCurrentTotal(holdings),
+        annualInvestment = 200 * 12 * holdings.length;
+
+    // Generate columns for the datagrid
+    stockCollection.forEach((stock, i) => {
+        const len = holdings[i].length,
+            lastHolding = holdings[i][len - 1],
+            ISIN = stock.ISIN,
+            tradingSymbol = stock.tradingSymbol;
+
+        dataGridData.push([
+            tradingSymbol,
+            ISIN,
+            Math.round(lastHolding / lastHoldingTotal * 100)
+        ]);
+    });
+
+    // Generate portfolio for risk score
+    const portfolio = {
+        name: 'PersonalPortfolio',
+        currency: 'EUR',
+        totalValue: lastHoldingTotal,
+        // Holdings with equal weights
+        holdings: getHoldings(100 / stockCollection.length)
+    };
+
+    // Creating chart and KPI options
+    const walletChartOptions = {
+        plotOptions: {
+            series: {
+                marker: {
+                    enabled: false
+                }
+            }
+        },
+        chart: {
+            height: 400
+        },
+        accessibility: {
+            description: 'displaying portfolio performance and investments'
+        },
+        rangeSelector: {
+            animate: false,
+            x: 0,
+            y: 0,
+            inputEnabled: false,
+            dropdown: 'never',
+            selected: 4
+        },
+        navigator: {
+            enabled: false
+        },
+        title: {
+            text: 'Portfolio performance'
+        },
+        yAxis: {
+            accessibility: {
+                description: 'price in euro'
+            }
+        },
+        tooltip: {
+            format: '<b>{x:%e - %b - %Y}</b><br/>' +
+                'Invested: €{points.1.y:,.2f}<br/>' +
+                'Holding: €{points.0.y:,.2f}<br/>' +
+                'Yield: {(multiply (divide (subtract points.0.y points.1.y)' +
+                ' points.1.y) 100):,.0f}%'
+        },
+        series: [{
+            name: 'Holding',
+            id: 'holding'
+        }, {
+            name: 'Invested',
+            id: 'invested',
+            className: 'dotted-line'
+        }],
+
+        responsive: {
+            rules: [{
+                condition: {
+                    maxWidth: 731
+                },
+                chartOptions: {
+                    chart: {
+                        height: 300
+                    },
+                    rangeSelector: {
+                        enabled: false
+                    }
+                }
+            }]
+        }
+
+    };
+
+    const riskScoreKPIOptions = {
+        chart: {
+            height: 186,
+            type: 'solidgauge',
+            className: 'hidden-title'
+        },
+        title: {
+            text: 'Risk score',
+            floating: true
+        },
+        pane: {
+            background: [{
+                backgroundColor: '#EEE',
```

Comment:
Does this apply at all? The charts all have `styledMode: true`, so color settings shouldn't work.